everyvoice/demo/app.py

```diff
@@ -91,12 +91,12 @@ def synthesize_audio(
         filelist=None,
         filelist_data=None,
         teacher_forcing_directory=None,
-        batch_size=1,
+        batch_size=16,
```

Comment:
I guess we should actually get this from the model. `text_to_spec_model.config.training.batch_size` or at least configurable from the cli

Reply:
We struggled with this one when Jordan first saw that only the first chunk ever got looked at, because of that formerly hard-coded `batch_size=1`.

I'm not sure we want to use the training batch size, because we will typically use the big computing resources at training time, but potentially much smaller computing resources at inference time. Take it from the CLI would make sense, with a default that is typically OK. Now that Jordan implemented re-joining chunks across batches, that default could be lower. Aidan, do you have an intuition on what is reasonable here on typical inference computing resources?

Reply:
See #701